_static/version_redirect.js

```diff
@@ -39,25 +45,29 @@ function stripVersionPath(path, versions) {
     var slash = path.indexOf('/', 1);
     if (slash != -1) {
         if (versions.indexOf(path.slice(1, slash)) != -1) {
-            path = path.slice(slash);
+            return [path.slice(1, slash), path.slice(slash)];
```

Comment:
Is this now a `split` function rather than a `strip`? This could also do with a comment, because I'm a bit lost among the slicing

Reply:
Right, this function has grown beyond its original purpose, so better to rename and comment.